server/sonar-db-core/src/main/resources/org/sonar/db/version/schema-h2.ddl

```diff
@@ -248,6 +249,12 @@ CREATE TABLE "QUALITY_GATE_CONDITIONS" (
   "UPDATED_AT" TIMESTAMP,
 );
 
+CREATE TABLE "ORG_QUALITY_GATES" (
+  "UUID" VARCHAR(40) NOT NULL PRIMARY KEY,
+  "ORGANIZATION_UUID" VARCHAR(40) NOT NULL,
+  "QUALITY_GATE_UUID" VARCHAR(40) NOT NULL
+);
+CREATE UNIQUE INDEX "UNIQ_ORG_QUALITY_GATES" ON "ORG_QUALITY_GATES" ("ORGANIZATION_UUID","QUALITY_GATE_UUID");
```

Comment:
I thought that the unique index should be on "QUALITY_GATE_UUID" ? 
Because as it is now, you are able to have a quality gate defined in 2 organizations, no ?